101-webapp-including-insights/azuredeploy.json

```diff
@@ -0,0 +1,142 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "webappName": {
+      "type": "string",
+      "defaultValue": "GEN-UNIQUE",
```

Comment:
this needs to be to use the uniqueString() function

Reply:
done